>From 3a4d8bba3ffd8a4c1e5a7c7318b49a1b68973dcb Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Wed, 4 Sep 2013 09:38:11 +0300
Subject: [PATCH 3/3] In debian-ispell.el Check debian-pkg-add-load-path-item
 before use
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 support/emacsen/debian-ispell.el |   14 ++++++++------
 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git a/support/emacsen/debian-ispell.el b/support/emacsen/debian-ispell.el
index 41e74e1..c9c2c18 100644
--- a/support/emacsen/debian-ispell.el
+++ b/support/emacsen/debian-ispell.el
@@ -458,17 +458,19 @@ LANG if any."
 		 ispell-program-name))
     )) ;; let and defun ends
 
-(add-hook 'after-init-hook 'debian-ispell-set-default-dictionary)
+;; (add-hook 'after-init-hook 'debian-ispell-set-default-dictionary)
+(eval-after-load "ispell" '(debian-ispell-set-default-dictionary))
 
 ;; ---------------------------------------------------------------------------
 ;; Make sure patched ispell.el is first in the loadpath if not already there
 ;; ---------------------------------------------------------------------------
 
-(let ((mypath (concat "/usr/share/"
-		      (symbol-name debian-emacs-flavor)
-		      "/site-lisp/dictionaries-common")))
-  (unless (member mypath load-path)
-    (debian-pkg-add-load-path-item mypath)))
+(when (fboundp 'debian-pkg-add-load-path-item)
+  (let ((mypath (concat "/usr/share/"
+			(symbol-name debian-emacs-flavor)
+			"/site-lisp/dictionaries-common")))
+    (unless (member mypath load-path)
+      (debian-pkg-add-load-path-item mypath))))
 
 ;; --------------------------------------------------------------------------
 ;; Set ispell-program-name consistently for all emacsen flavours. Prefer
-- 
1.7.10.4

